src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2555,3 +2555,148 @@
 type: command
 short-summary: List out all available versions of Bicep CLI.
 """
+
+helps['stacks'] = """
+type: group
+short-summary: Manage deployment stacks at subscription or resource group scope.
+"""
+
+helps['stacks sub create'] = """
+type: command
+short-summary: Create a deployment stack at subscription scope
+examples:
+  - name: Create a deployment stack using template file.
+    text: az stacks sub create --name "StackName" --update-behavior "detachResources" --template-file simpleTemplate.json --location "westus2" --description "description"
+  - name: Create a deployment stack with parameter file.
+    text: az stacks sub create --name "StackName" --update-behavior "detachResources" --template-file simpleTemplate.json --param-file simpleTemplateParams.json --location "westus2" --description "description"
+  - name: Create a deployment stack with template spec
+    text: az stacks sub create --name "StackName" --update-behavior "detachResources" --template-spec "TemplateSpecResourceIDWithVersion" --location "westus2" --description "description"
+"""
+
+helps['stacks sub list'] = """
+type: command
+short-summary: Show all deployment stacks in subscription
```

Comment:
```suggestion
short-summary: List all deployment stacks in subscription
```

Reply:
It is recommended to use `List`, because `show` is sometimes used to describe GET operation in the help description of CLI, which is easy to confuse users